iosApp/iosApp/ui/textFields.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+
+struct EditText: View {
+    @Binding var input: String
```

Comment:
Could you explain why u used @Binding and not @State and what the difference is?

Reply:
@Binding creates a two-way connection between view & property, while @State is a single source of truth. From what I could tell @State is usually used in view models only